cypress/integration/addtodo_spec.js

```diff
@@ -1,11 +1,35 @@
-describe('My First Test', () => {
-  it('Adds a new to-do to the list', () => {
-    cy.visit('./index.html')
-    cy.get("#todo-input").type("Wash dishes")
-    cy.contains("Add to list").click()
+describe("My First Test", () => {
+  it("adds a new to-do to the list", () => {
+    cy.get("input, textarea").clear().type("Wash dishes");
```

Comment:
> `"input, textarea"`

This way the original tests will still pass for submissions that switch to a multiline textarea for the bonus.